src/Uno.UI/ts/WindowManager.ts

```diff
@@ -276,6 +276,9 @@ namespace Uno.UI {
 				element.classList.value = classes;
 			}
 
+			// Disable pointer events by default to match HitTestVisibilityProperty default value
+			element.style.pointerEvents = "none";
```

Comment:
WOuld be nice to put it here instead https://github.com/unoplatform/uno/blob/master/src/Uno.UI/WasmCSS/Uno.UI.css#L23 :)